tests/tendermint/test_lib.py

```diff
@@ -39,10 +40,12 @@ def test_asset_is_separated_from_transaciton(b):
                             asset=asset)\
                     .sign([alice.private_key])
 
-    b.store_transaction(tx)
+    tx_dict = copy.deepcopy(tx.to_dict())
```

Comment:
Is `deepcopy` really needed here? I assume `store_bulk_transactions` will just write the contents of `tx` to mongo. That should leave `tx` unchanged, and you can still perform the comparison in the last assert.

What breaks if you skip the `deepcopy`?